### PR TITLE
5 random fixes (#2)

### PR DESCRIPTION
**Pull request by pepyakin**
In the spirit of https://github.com/paritytech/substrate/pull/435

It is better to check each commit separately.

### CODE REVIEW

**Comment by gavofyork** [comment]
the double-spaces are in legal text and (i think) there on purpose.

File: demo/runtime/src/lib.rs [hunk 1]
```diff
@@ -8,11 +8,11 @@
 
 // Substrate Demo is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. ee the
```
Comment (gavofyork) [inline demo/runtime/src/lib.rs]: and this is plain wrong :)

**Comment by gavofyork** [comment]
though there are a few correct removals of double-spacing

File: demo/runtime/src/lib.rs (hunk 1 above)
Comment (pepyakin) [reply demo/runtime/src/lib.rs]: It should have happened...

**Comment by pepyakin** [comment]
Oh, nice to know, was wondering if that was on purpose! will revert the removal of the double spaces from the legal.

> though there are a few correct removals of double-spacing

You mean, in the strings?

**Comment by pepyakin** [comment]
Removed: https://github.com/paritytech/substrate/pull/623/commits/938706afa8f3f98ea29a4d6959eca2c8982f01ac

**Pull request merged**